.github/workflows/build.yml

```diff
@@ -23,10 +27,29 @@ jobs:
           distribution: 'zulu'
           java-version: 19
 
-      - name: build
+      - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
-        with:
-          arguments: build --full-stacktrace
+
+      - name: Build Multi-platform Modules
+        run: |
+          ./gradlew \
+            :xef-core:${{ matrix.platform }}Test \
```

Comment:
We're excluding some other checks like formatting with this check.

Reply:
Added to the pipeline